docs/Video Streaming for Navigation Apps/Audio Streaming/index.md

```diff
@@ -11,11 +11,43 @@ To stream audio from a SDL app, use the @![iOS]`SDLStreamingMediaManager`!@@![an
 @![iOS]
 ## Audio Stream Lifecycle
 Like the lifecycle of the video stream, the lifecycle of the audio stream is maintained by the SDL library. When you receive the `SDLAudioStreamDidStartNotification`, you can begin streaming audio.
+!@
 
 ### Audio Stream Manager
-The `SDLAudioStreamManager` will help you to do on-the-fly transcoding and streaming of your files in mp3 or other formats, or prepare raw PCM data to be queued and played.
+The @![iOS]`SDLAudioStreamManager`!@@![android]`AudioStreamManager`!@ will help you to do on-the-fly transcoding and streaming of your files in mp3 or other formats, or prepare raw PCM data to be queued and played.
+
+@![android]
+### Starting the Audio Manager 
+To stream audio, we call `sdlManager.getAudioStreamManager().start()` which will start the manager. When that callback returns successful, you call `sdlManager.getAudioStreamManager().startAudioStream()`. ONce this callback is successful you can push your audio source to the `AudioStreamManager`.
+
+```java
+if (sdlManager.getAudioStreamManager() != null) {
```

Comment:
Im not sure I agree with this change, depending on where the developer uses this they might not want to return from their method if the manager is null

Reply:
It's just example code, you could do something like

```java
// Check if the manager is null, this could happen because of xyz
if (sdlManager.getAudioStreamManager() == null) {
    // Handle the failure
    return null
}
```

If you still disagree, that's fine, it's just a suggestion for code readability.